### DISCUSSION TITLE
AppStore (Admin only access) [Apr 17]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5532] [Discord Threads]
> 🙋‍♂️ **Isma**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### Appstore is now part of account settings panel for admin.    
#### https://docs.nocodb.com/setup-and-usages/account-settings#app-store  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1086313511187128470)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
